### DISCUSSION TITLE
tutorial/use/skinswitch

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#24] [General]
# tutorial/use/skinswitch

http://localhost:5173/tutorial/use/skinswitch

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by long36708** [comment]
开始还以为是使用了Context多实例，仔细一看，是card里还定义了一个number 的state啊！
